cvat-core/src/annotations-objects.ts

```diff
@@ -65,12 +65,14 @@ class Annotation {
     protected history: any;
     protected groupColors: Record<number, string>;
     public serverID: number | null;
+    public jobID: number;
     protected parentID: number | null;
     protected dimension: DimensionType;
     public group: number;
     public label: Label;
     public frame: number;
     private _removed: boolean;
+    protected isGroundTruth: boolean;
```

Comment:
Do we really need both jobID and isGroundTruth?

Reply:
Yes, we can compare jobID to GT job id all the time, but I think using such parameter makes everything much simplier